src/collections/_documentation/platforms/php/index.md

```diff
@@ -68,12 +68,29 @@ Sentry\init([
 ### Extra Context
 In addition to the structured context that Sentry understands, you can send arbitrary key/value pairs of data which the Sentry SDK will store alongside the event. These are not indexed, and the Sentry SDK uses them to add additional information about what might be happening.
 
+You can add extra data to all events:
+
 ```php
 Sentry\configureScope(function (Sentry\State\Scope $scope): void {
   $scope->setExtra('character_name', 'Mighty Fighter');
+  
+  // to set multiple key-value pairs at once:
+  $scope->setExtras([
+    'character_name' => 'Mighty Fighter',
+    'sidekick' => 'Awesome Possum'
+  ]);
 });
 ```
 
+or per event:
+
+```php
+Sentry\withScope(function (Sentry\State\Scope $scope): void {
+  $scope->setExtra('character_name', 'Mighty Fighter');
+  // or $scope->setExtras([ ... ])
+  Sentry\captureException(new \Throwable('my error'))
```

Comment:
`\Throwable` in PHP is an interface, so it cannot be thrown. Please change this code to `new \Exception('...')`

Reply:
I hear you re: top-level vs scope. Above my pay-grade though.

Fixed `Throwable`, here and below. Got that from [here](https://github.com/getsentry/sentry-docs/blob/e6c1252b13e7d0957ad2dcee36a96ae496ae0619/src/collections/_documentation/enriching-error-data/with-scope/php.md#L7). Should I change it there as well?

Reply:
Yes please, that code would throw a compile error otherwise as it’s not valid

Reply:
✔️

Reply:
I would ping @HazAT for the question about the global call vs `Scope` method

Reply:
Yup, pinged him in slack before I pushed. Just waiting to hear back.